colorcet/__init__.py

```diff
@@ -113,25 +113,23 @@ def register_cmap(name, cmap):  # type: ignore[no-redef]
 
 
 def rgb_to_hex(r: int, g: int, b: int) -> str:
-    return f"#{r:02x}{g:02x}{b:02x}"
+    return '#%02x%02x%02x' % (r,g,b)
```

Comment:
Not sure why this change is an improvement.

Reply:
f strings are easier to understand for me and are more modern. But I've reversed the change tho

Reply:
I agree with using f-strings, but as written the change is to _remove_ the f-string. Maybe I'm confused somehow?
<img width="304" alt="image" src="https://github.com/user-attachments/assets/ff1ada02-0557-4feb-acbe-53a5d56c697d" />